models/index.js

```diff
@@ -0,0 +1,60 @@
+let datastore = require('@google-cloud/datastore');
+let config = require('../config/config');
+
+let client = datastore({
+  projectId: config.projectId,
+  keyFilename: config.googleCloudKey
+});
+
+module.exports.client = client;
+
+module.exports.savePayload = (doc, successCallback, errorCallback) => {
+  validateProduct(doc)
+
+  if (!validateProduct(doc)) {
+    return errorCallback(doc.errors)
+  }
+
+  const collectorKey = client.key('Collector');
+  const now = new Date()
+
+  doc['created_at'] = now
+  doc['updated_at'] = now
+
+  client.save({
+    key: collectorKey,
+    data: doc
+  }, function (err, res) {
+    if (err) {
+      errorCallback(err);
+    } else {
+      successCallback(collectorKey);
```

Comment:
should I pass the recently created collector document to the successCallback, or does the collectorKey suffice? It contains the id and the kind attributes.